Data-Structure-Stacks.md

```diff
@@ -0,0 +1,60 @@
+# Data Structure Stacks
+
+`Stack` is a linear data structure which follows a particular order in which the operations are performed. The order may be 
+LIFO(Last In First Out) or FILO(First In Last Out).
```

Comment:
I have never encountered the term `FILO` in standard text. I only know of `LIFO`. Can you cite some reference to this? Otherwise, remove it please.